server/portal/settings/settings.py

```diff
@@ -726,8 +727,12 @@ def gettext(s): return s  # noqa:E731
 """
 SETTINGS: RECAPTCHA
 """
-RECAPTCHA_SECRET_KEY = settings_secret._RECAPTCHA_SECRET_KEY
-RECAPTCHA_SITE_KEY =  settings_secret._RECAPTCHA_SITE_KEY
+if settings_secret._RECAPTCHA_SECRET_KEY and settings_secret._RECAPTCHA_SITE_KEY:
+    RECAPTCHA_SECRET_KEY = settings_secret._RECAPTCHA_SECRET_KEY
+    RECAPTCHA_SITE_KEY =  settings_secret._RECAPTCHA_SITE_KEY
+else:
+    RECAPTCHA_SECRET_KEY = unit_test_settings.RECAPTCHA_SECRET_KEY
+    RECAPTCHA_SITE_KEY =  unit_test_settings.RECAPTCHA_SITE_KEY
```

Comment:
Rather than depend on unit test settings, we want to make this fully backwards compatible such that if no recaptcha secret or site key is defined, the feature is simply disabled.

Reply:
gotcha, basically this will pass the Recaptcha function and let the user know that there is a problem with the keys and report to us but I can do something that hides and disable the Recaptcha function.
![recaptcha_test](https://user-images.githubusercontent.com/34716439/146404080-e26000de-46e2-442c-be87-23861b2e2d90.png)
t

Reply:
Right, we want to keep these values and feature backwards compatible, such that if no keys are provided in `settings_secret.py`, the feature is simply not rendered:

`RECAPTCHA_SECRET_KEY = getattr(settings_secret, '_RECAPTCHA_SECRET_KEY', None)`

Additionally, we want to conditionally evaluate the render function on the frontend and the recaptcha evaluation on the backend depending on whether the key is set